src/i18n.js

```diff
@@ -1,17 +1,14 @@
-const messages = {
-	en: {
-		'wikipedia': 'Wikipedia'
-	},
-	fr: {
-		'wikipedia': 'Wikipédia'
-	},
-	es: {
-		'wikipedia': 'Wikipedia'
+import fallbackMessage from '../i18n/en.json'
+const msg = ( lang, key ) => {
+	let messages
+
+	try {
+		messages = require( `../i18n/${lang}.json` )
```

Comment:
Is this going to load the file every time the `msg` function is called?

Reply:
nope, the file (object) is being stored within the internal code.

Reply:
sorry, got the wrong understanding here, true, we should make it available once the require is called for the next call used.

Reply:
done, and please check it again.